content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/_cq_design_dialog/.content.xml

```diff
@@ -136,6 +136,30 @@
                         jcr:primaryType="nt:unstructured"
                         sling:resourceType="granite/ui/components/coral/foundation/include"
                         path="/mnt/overlay/cq/gui/components/authoring/dialog/style/tab_design/styletab"/>
+                    <features
```

Comment:
@Shivanshmathur - I would rather keep all the properties together in one tab. It's a bit unfortunate that the node name of the first tab is "sizes". We can still rename it's jcr:title however to "Title" and add these options there. I think the teaser component design dialog is a good example to follow.

Reply:
Noted and Done.